README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/GameAnalytics/ansible-carbon-relay-ng.svg?token=psJ7kzCPVMkCYuYpVEsy&branch=master)](https://travis-ci.com/GameAnalytics/ansible-carbon-relay-ng)
+[![Build Status](https://travis-ci.org/GameAnalytics/ansible-carbon-relay-ng.svg?token=psJ7kzCPVMkCYuYpVEsy&branch=master)](https://travis-ci.org/GameAnalytics/ansible-carbon-relay-ng)
```

Comment:
Do we use the same token for `.org`?

Reply:
Do we need to change to `.org`? Looks like travis is migrating all to `.com` and recommends putting everything there now